signac/contrib/indexing.py

```diff
@@ -44,13 +52,16 @@ def _is_blank_module(module):
         return not bool(file.read().strip())
 
 
+# this class is deprecated
 class BaseCrawler(object):
```

Comment:
Is there a reason we don't just deprecate all of the classes directly, rather than going through and deprecating methods?

Reply:
It's a work-around for this bug: https://github.com/briancurtin/deprecation/issues/43 

So, unfortunately necessary for classes that we inherit from ...

Reply:
Wonderful. @tommy-waltmann that helps you then, you can ignore about half my comments :)